Makefile

```diff
@@ -36,9 +36,8 @@ package-%:
 	@rm -f debian/changelog
 	dch --create --package delphix-platform -v $(VERSION) \
 			"Automatically generated changelog entry."
-	# Add a file which can be queried to find out the platform on a
-	# running engine.
-	echo "$*" >platform
+	sed "s/@@PLATFORM@@/$*/" var/lib/delphix-appliance/platform.in \
```

Comment:
Any reason not to use TARGET_PLATFORM here too?

Reply:
+1 .. I also noticed the inconsistency, but didn't bother to mention it.